lib/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -21,6 +22,69 @@ const meta = {
 	fixable: true,
 };
 
+/** @type {(node: import('css-tree').CssNode) => node is import('css-tree').Value} */
+const isValueNode = (node) => {
+	return node.type === 'Value';
+};
+
+/** @type {(node: import('css-tree').CssNode) => node is import('css-tree').FunctionNode} */
+const isFunctionNode = (node) => {
+	return node.type === 'Function';
+};
+
+/** @type {(node1: import('css-tree').CssNode[], node2: import('css-tree').CssNode[]) => boolean} */
+const isEqualValueNodes = (nodes1, nodes2) => {
+	if (nodes1.length !== nodes2.length) {
+		return false;
```

Comment:
[ask] Line 38 was not covered by tests, but could you add a test case easily to cover the line?